---
layout: page
image:
  feature: banner_publications-mini.jpg
---

## 2014
<hr />

* T.A. Hopf#, C.P.I. Sch√§rfe#, J.P.G.L.M. Rodrigues#, A.G. Green, O. Kohlbacher, C. Sander\*, A.M.J.J. Bonvin* and D.S. Marks*.
"Sequence co-evolution gives 3D contacts and structures of protein complexes."
eLife. 10.7554/eLife.03430, Advanced Online Publication (2014)
<a href="http://elifesciences.org/content/early/2014/09/25/eLife.03430" class="btn btn-success">Link</a>

* M. Rey, V. Sarpe, K. Burns, J. Buse, Ch.A.H. Baker, M. van Dijk, L. Wordeman, A.M.J.J. Bonvin and D. Schriemer.
"Mass Spec Studio for Integrative Structural Biology."
Structures. Advanced Online Publication (2014).

*  M. Trellet, A.S.J. Melquiond and A.M.J.J. Bonvin.
"Information-driven modelling of protein-peptide complexes."
Methods in Molecular Biology. Ed. Peng Zhou. Humana Press Inc. In press.

*  J.P.G.L.M Rodrigues, E. Karaca and A.M.J.J. Bonvin.
"Information-driven structural modelling of protein-protein interactions."
Methods in Molecular Biology: Molecular Modelling of Proteins. Ed. Andreas Kokul. Humana Press Inc. In press.

*  W.F. Vranken, G.W. Vuister and A.M.J.J. Bonvin.
"NMR-based modelling and refinement of protein 3D structures."
Methods in Molecular Biology: Molecular Modelling of Proteins. Ed. Andreas Kokul. Humana Press Inc. In press.

*  P.L. Kastritis, J.P.G.L.M Rodrigues, G.E. Folkers, R. Boelens and A.M.J.J. Bonvin.
"Proteins feel more than they see: Fine-tuning of binding affinity by properties of the non-interacting surface."
J. Mol. Biol. 426, 2632-2652 (2014). Access the recommendation on F1000Prime

*  J.P.G.L.M Rodrigues and A.M.J.J. Bonvin
"Integrative computational modeling of protein interactions."
FEBS J., 281, 1988-2003 (2014).

*  P.L. Kastritis, J.P.G.L.M. Rodrigues and A.M.J.J. Bonvin
"HADDOCK2P2I: A robust biophysical model for predicting the binding affinity of protein-protein interaction inhibitors"
J. Chem. Info. Model. 54, 826-836 (2014).

*  H. van Ingen and A.M.J.J. Bonvin
"Information-driven modeling of large macromolecular assemblies using NMR data."
J. Magn. Reson. 241, 103-114 (2014).

*  J. Snijder, R.J. Burnley, A. Wiegard, A.S.J. Melquiond, A.M.J.J. Bonvin, I.M. Axmann, A.J.R. Heck
"Insight into cyanobacterial circadian timing from structural details of the KaiB-KaiC interaction."
Proc. Natl. Acad. Sci. USA. 111, 1379-1383 (2014).

*  G.C.P. van Zundert and A.M.J.J. Bonvin.
"Modeling protein-protein complexes using the HADDOCK webserver."
Methods in Molecular Biology: Protein Structure Prediction. Ed. Daisuke Kihara. Humana Press Inc., 163-179 (2014).

*  M.F. Lensink, I.H. Moal, P.A. Bates, P.L. Kastritis, A.S.J. Melquiond, E. Karaca, C. Schmitz, M. van Dijk, A.M.J.J. Bonvin, M. Eisenstein, B. Jimenez-Garcia, S. Grosdidier, S.A Solernou, L. Perez-Cano, C. Pallara, J. Fernandez-Recio, J. Xu, P. Muthu, K. Praneeth Kilambi, J.J. Gray, S. Grudinin, G. Derevyanko, J.C. Mitchell, J. Wieting, E. Kanamori, Y. Tsuchiya, Y. Murakami, J. Sarmiento, D.M. Standley, M. Shirota, K. Kinoshita, H. Nakamura M. Chavent, D.W. Ritchie, H. Park, J. Ko, H. Lee, C. Seok, Y. Shen, D. Kozakov, S. Vajda, P. J. Kundrotas, I.A. Vakser, B.G. Pierce, H. Hwang, T. Vreven, Z. Weng, I. Buch, E. Farkash, H.J. Wolfson, M. Zacharias, S. Qin, H.-X. Zhou, S.-Y. Huang, X. Zou, J.A. Wojdyla, C. Kleanthous and S.J. Wodak.
"Blind Prediction of Interfacial Water Positions in CAPRI."
Proteins: Struc. Funct. & Bioinformatics 82, 620-632 (2014).

## 2013
<hr />

* K.M. Varney#, A.M.J.J. Bonvin#, M. Pazgier#, J. Malin#, W. Yu#, E. Ateh, T. Oashi, J. Huang, W. Lu, M. Diepeveen-de Buin, J. Bryant, E. Breukink, A.D. MacKerell Jr. and E.P.H. de Leeuw
"Turning Defense into Offense: Development of Defensin Mimetics as Novel Antibiotics targeting Lipid II."
PloS Pathogens, 9, e1003732. doi:10.1371/journal.ppat.1003732 (2013).

* G. van der Schot, Z. Zhang, R. Vernon, Y. Shen, W.F. Vranken, D. Baker, A.M.J.J. Bonvin* and O.F. Lange*
"Improving 3D structure prediction from chemical shift data."
J. Biomol. NMR., 57, 27-35 (2013).

* A.M.J.J. Bonvin
"Coming to peace with protein complexes? 5th CAPRI evaluation meeting, April 17-19th 2013, Utrecht."
Proteins: Struc. Funct. & Bioinformatics Editorial, CAPRI special issue. 81, 20173-2074 (2013).

* J.P.G.L.M. Rodrigues, A.S.J. Melquiond, E. Karaca, M. Trellet, M. Van Dijk, G.C.P. Van Zundert, C. Schmitz, S.J. de Vries, A. Bordogna, L. Bonati, P.L. Kastritis and A.M.J.J. Bonvin
"Defining the limits of homology modelling in information-driven protein docking"
Proteins: Struc. Funct. & Bioinformatics, 81, 2119-2128 (2013).

* R. Morretti et al.
"Community-wide Evaluation of Methods for Predicting the Effect of Mutations on Protein-Protein Interactions."
Proteins: Struc. Funct. & Bioinformatics, 81, 1980-1987 (2013).

* J. Janin and A.M.J.J. Bonvin
"Protein-Protein Interactions."
Curr. Opin. Struct. Biol. Editorial, special issue on protein-protein interactions. 23, 859-861 (2013).

* P.L. Kastritis and A.M.J.J. Bonvin
"Molecular origins of binding affinity: Seeking the Archimedean point."
Curr. Opin. Struct. Biol., 23, 868-877 (2013).

* D.M. Dias, J.P.G.L.M. Rodrigues, N.S. Domingues, A.M.J.J. Bonvin and M.C.A. Castro.
"Unveiling the Interaction of Vanadium Compounds with Human Serum Albumin using 1H STD NMR and Computational Docking Studies."
Eur. J. Inorg. Chem., 26, 4619-4626 (2013).

* E.A.W. van der Cruijsen, D. Nand, M. Weingarth, A, Prokofyev, S, Hornig, A.M.J.J. Bonvin, S. Becker, R.E. Hulse, E. Perozo, O. Pongs, M. Baldus.
"The importance of the lipid-pore loop interface for potassium channel structure and function."
Proc Natl. Acad. Sci. USA, 110, 13008-13013. (2013).

* M. van Dijk, K. Visscher, P.L. Kastritis and A.M.J.J. Bonvin.
"Solvated protein-DNA docking using HADDOCK."
J. Biomol. NMR, 56, 51-63 (2013).

* E. Karaca and A.M.J.J. Bonvin.
"On the usefulness of Ion Mobility Mass Spectrometry and SAXS data in scoring docking decoys."
Acta Cryst. D., D69, 683-694 (2013).

* M. Weingarth, A. Prokofyev, E. van der Cruijsen, D. Nand, A.M.J.J. Bonvin, O. Pongs and M. Baldus
"Structural determinants and functional aspects of specific lipid binding to potassium channels."
J. Am. Chem. Soc., 35, 3983-3988 (2013).

* S. Alex, K. Lange, T. Amolo, J.S. Grinstead, E. Szalowska, A. Koppen, K. Mudde, D. Haenen, S. Al-Lahham, H. Roelofsen, R. Houtman, B. van der Burg, A.M.J.J. Bonvin, E. Kalkhoven, M. Muller, G. Hooiveld and S. Kersten.
"Short chain fatty acids stimulate Angptl4 synthesis in human colonocytes by selective PPAR modulation."
Mol. Cell. Biol., 33, 1303-1316 (2013).

* D.L. Hulsik, Y.Y. Liu, N.M. Strokappe, S. Battella, M. El Khattabi, L.E. McCoy, A. Hinz, M. Hock, A.M.J.J. Bonvin, J.P.M. Langedijk, D. Davis, A. Forsman Quinley, M. Aasa-Chapman, R.A. Weiss, C.T. Verrips, W. Weissenhorn and L. Rutten
"A gp41 MPER-specific Llama VHH Requires a Hydrophobic CDR3 for Neutralization but not for Antigen Recognition."
PloS Pathogens, 9, e1003202 (2013).

* M. Trellet, A.S.J. Melquiond and A.M.J.J. Bonvin.
"A Unified Conformational Selection and Induced Fit Approach to Protein-Peptide Docking.
PLoS ONE, 8(3) e58769 (2013).

* R. Dagil, C O'Shea, A. Nykaer, A.M.J.J. Bonvin and B.B. Kragelund
"Gentamicin Binds to Megalin as a Competitive Inhibitor and Utilizes the Common Ligand Binding Motif of Complement Type Repeats."
J. Biol. Chem, 288, 4424-4435 (2013).

* E. Karaca and A.M.J.J. Bonvin
"Advances in integrated modelling of biomolecular complexes."
Methods, 59, 372-381 (2013).

* P.L. Kastritis and A.M.J.J. Bonvin
"On the binding affinity of macromolecular interactions: daring to ask why proteins interact"
J. R. Soc. Interface, 10, doi: 10.1098/rsif.2012.0835 (2013).

* A.M.J.J. Bonvin, M. van Dijk, E. Karaca, P.L. Kastritis, A.S.J. Melquiond, C. Schmitz and S.J. de Vries
"HADDOCK"
In Encyclopedia of Biophysics, Ed. G.C.K. Roberts, Springer-Verlag Berlin Heidelberg (2013).

* P.L. Kastritis, K.M. Visscher, A.D.J. van Dijk and A.M.J.J. Bonvin
"Solvated docking using Kyte-Doolittle-based water propensities."
Proteins: Struc. Funct. & Bioinformatic., 81, 510-518 (2013).

## 2012
<hr />

* S. Gradmann, C. Ader, I. Heinrich, D. Nand, M. Dittmann, A. Cukkemane, M. van Dijk, A.M.J.J. Bonvin, M. Engelhard and M. Baldus
"Rapid Prediction of multi-dimensional NMR data sets."
J. Biomol. NMR, 54, 377-387 (2012).

* J. Hennig, S. de Vries, K.D.M. Hennig, L. Randles, K.J. Walters, M. Sunnerhagen and A.M.J.J. Bonvin
MTMDAT-HADDOCK: High-throughput, protein complex structure modeling based on limited proteolysis and mass spectrometry."
BMC Struc. Biol., 12, 29 (2012).

* S.J.L. van Wijk, A.S.J. Melquiond, S.J. de Vries, H.Th.M Timmers and A.M.J.J. Bonvin
"Dynamic control of selectivity in E2-E3 interactions via a salt-bridge network revealed by an ASP to GLU substitution." 
PLoS Comp. Biol., 8(11), e1002754 (2012).

* T.A. Wassenaar, M. van Dijk, N. Loureiro-Ferreira, G. van der Schot, S.J. de Vries, C. Schmitz, J. van der Zwan, R. Boelens, A. Giachetti, L. Ferella, A. Rosato, I. Bertini, T. Herrmann, H.R.A. Jonker, A. Bagaria, V. Jaravine, P. Guntert, H. Schwalbe, W.F. Vranken, J.F. Doreleijers, G. Vriend, G.W. Vuister, D. Franke, A. Kikhney, D.I. Svergun, R. Fogh, J. Ionides, E.D. Laue, C. Spronk, S. Jurka, M. Verlato, S. Badoer, S. Dal Pra, M. Mazzucato, E. Frizziero and A.M.J.J. Bonvin
"WeNMR: Structural Biology on the Grid."
J. Grid. Comp., 10, 743-767 (2012).

* T.A. Wassenaar, S.J.L. de Vries, H. Bekker and A.M.J.J. Bonvin
"SQUEEZE-E: The optimal solution for molecular simulations with periodic boundary conditions."
J. Chem. Theo. Comput., 8, 3618-3627 (2012).

* M. van Dijk, T.A. Wassenaar and A.M.J.J. Bonvin
"A flexible, grid-enabled web portal for GROMACS molecular dynamics simulations"
J. Chem. Theo. Comput., 8, 3463-3472 (2012).

* P.L. Kastritis and A.M.J.J. Bonvin
"Predicting and Dissecting High-order Molecular Complexity by Information-driven Biomolecular Docking"
In: Antimicrobial Drug Discovery. Emerging Strategies Adv. Mol. Cell. Microbiol., Eds G. Tegos and E. Mylokanis, CABI, 22, 232-246 (2012).

* M. Weingarth, C. Ader, A.S.J. Melquiond, D. Nand, O. Pongs, S. Becker, A.M.J.J. Bonvin and M. Baldus.
"Supramolecular structure of membrane-associated polypeptides by combining solid-state NMR and Molecular Dynamics simulations."
Biophys. J., 103, 29-37 (2012).

* J.P.G.L.M. Rodrigues, M. Trellet, C. Schmitz, P.L. Kastritis, E. Karaca, A.S.J. Melquiond and A.M.J.J. Bonvin
"Clustering biomolecular complexes by residue contacts similarity."
Proteins: Struc. Funct. & Bioinformatic, 80, 1810-1817 (2012).

* C. Schmitz, A.S.J. Melquiond, S.J. de Vries, E. Karaca, M. van Dijk, P.L. Kastritis and A.M.J.J. Bonvin
"Protein-protein docking with HADDOCK"
In: NMR in Mechanistic Systems Biology. Ed. I. Bertini, K.S. McGreevy and G. Parigi, Wiley-Blackwell, 512-535.

* A. Rosato, J. M. Aramini, C. Arrowsmith, A. Bagaria, D. Baker, A. Cavalli, J.F. Doreleijers, A. Eletsky, A. Giachetti, P. Guerry, A. Gutmanas, P. Guntert, Y. He, T. Herrmann, Y.J. Huang, V. Jaravine, H.R.A. Jonker, M.A. Kennedy, O.F. Lange, G. Liu, T.E. Malliavin, R. Mani, B. Mao, G.T. Montelione, M. Nilges, P. Rossi, G. van der Schot, H. Schwalbe, T.A. Szyperski, M. Vendruscolo, R. Vernon, W.F. Vranken, S.J. de Vries, G.W. Vuister, B. Wu, Y. Yang and A.M.J.J. Bonvin
"Blind testing of routine, fully automated determination of protein structures from NMR data."
Structure, 20, 227-236 (2012).

* P.L. Kastritis, A.D.J. van Dijk and A.M.J.J. Bonvin
"Explicit Treatment of Water Molecules in Data-Driven Protein-Protein Docking: The Solvated HADDOCKing Approach"
Methods in Molecular Biology 819, Part 5, 355-374 (2012)

* A.S.J. Melquiond, E. Karaca, P.L. Kastritis and A.M.J.J. Bonvin
"Next challenges in protein-protein docking: From proteome to interactome and beyond."
WIREs Computational Molecular Science, 2, 642-651 (2012).